vmm/src/config.rs

```diff
@@ -2100,6 +2128,51 @@ impl TpmConfig {
     }
 }
 
+impl LandlockConfig {
+    pub const SYNTAX: &'static str = "Landlock parameters \
+        \"path=<path/to/{file/dir}>,flags={r,w,x}\". Both path and flags options are required.\"";
+
+    pub fn parse(landlock_rule: &str) -> Result<Self> {
+        let mut parser = OptionParser::new();
+        parser.add("path").add("flags");
+        parser
+            .parse(landlock_rule)
+            .map_err(Error::ParseLandlockRules)?;
+
+        let path = parser
+            .get("path")
+            .map(PathBuf::from)
+            .ok_or(Error::ParseLandlockMissingFields)?;
+        let mut path_flags: u8 = 0;
+        if let Some(flags) = parser.get("flags") {
+            for c in flags.chars() {
+                match c {
```

Comment:
I'd suggest to be conservative and only allow one character of each to not accept cases like "rwxxxxrr".

Reply:
I will address this

Reply:
done